CHANGELOG.markdown

```diff
@@ -1,3 +1,7 @@
+5.6.1 [2023.mm.dd]
+------------------
+* Provide instances for `Swap` and `Assoc` type-classes from `assoc` package.
```

Comment:
Minor English-language suggestion:

```suggestion
* Provide instances for the `Swap` and `Assoc` type classes from the `assoc` package.
```

Also, it is worth mentioning here that these instances aren't brand-new: they were migrated over from `assoc`.